roles/installer/templates/tower_postgres.yaml.j2

```diff
@@ -60,6 +60,14 @@ spec:
             - name: postgres
               mountPath: '{{ tower_postgres_data_path | dirname }}'
               subPath: '{{ tower_postgres_data_path | dirname | basename }}'
+{% if tower_node_selector %}
+      nodeSelector:
+        {{ tower_node_selector | indent(width=8) }}
+{% endif %}
+{% if tower_tolerations %}
```

Comment:
Same here should be `tower_postgres_tolerations`

Reply:
Fixed it in d4d9d2a.